docs/ch-rob.md

```diff
@@ -0,0 +1,3 @@
+**Name**: Chad Voelker  
+**Occupation**: Key Presser  
+**Hometown**: The sprawling metropolis of Bonduel, WI  
```

Comment:
@jci-trawicki  It has grown by about 20% since I left... everything's relative. :tongue: